api/log/log.go

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package log
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"k8s.io/klog/v2"
+
+	"github.com/kiegroup/kogito-serverless-operator/container-builder/util"
+)
+
+// Log --.
+var Log Logger
+
+func init() {
+	Log = Logger{
+		delegate: klog.NewKlogr().WithName(util.ComponentName),
+	}
+}
+
+// Injectable identifies objects that can receive a Logger.
+type Injectable interface {
```

Comment:
We can remove this too. Ideally we can use loggers directly by calling a `log` interface.

Reply:
Removed and refactored where applicable.